src/components/FormInput/index.tsx

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+
+function FormInput() {
+  return (
+    <>
+      <StyledFormInput type="text" />
+    </>
+  );
+}
+
+export default FormInput;
+
+const StyledFormInput = styled.input`
+  display: flex;
+  width: 80%;
+  margin: 0 auto;
+  padding: 16px;
+  color: white;
```

Comment:
```suggestion
  color: ${({ theme }) => theme.color.white};
```
white 라는 컬러 theme에 적용되어 있어서 이렇게 사용하시는 게 좋을 것 같아요

Reply:
수정하겠습니다 !